release-process-documentation/release-team-guides/ci-signal-playbook.md

```diff
@@ -1,41 +1,141 @@
-- I need the ability to add a milestone to issues, so I must be a member of the [kubernetes-pm github team](https://github.com/orgs/kubernetes/teams/kubernetes-pm/members)
+- Signal lead needs the ability to add a milestone to issues, so must be a member of the [kubernetes-pm github team](https://github.com/orgs/kubernetes/teams/kubernetes-pm/members)
   - The process to join this group is in [our community project-managers page](https://github.com/kubernetes/community/blob/master/project-managers/README.md#joining-the-group)
-- I need to understand what tests matter and generally how our testing infra is wired together
-  - I can ask previous CI Signal leads for advice
-  - I can ask sig-testing for guidance
+- Signal lead need to understand what tests matter and generally how our testing infra is wired together
+  - He/she can ask previous CI Signal leads for advice
+  - He/she can ask SIG-Testing for guidance
 
-## Status Quo
+## Overview of tasks across release timeline
 
-Currently, here's what I'm doing as the CI Signal role for release 1.y:
+For any release, its schedule and activities/deliverable for each week will be published in the release directory, e.g: [1.11 schedule](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.11/release-1.11.md#timeline). This section talks about specific CI Signal lead deliverable for each milestone in the release cycle.
 
-### Checking test dashboards
+### Pre Feature Freeze
 
-I check [sig-release-master-blocking](https://k8s-testgrid.appspot.com/sig-release-master-blocking) daily.
+Here are some good early deliverables from the CI Signal lead between start of the release to feature freeze.
+- Maintain a master tracking sheet and keep it up-to-date with issues tracking any test failure/flake - [Sample sheet](https://docs.google.com/spreadsheets/d/1j2K8cxraSp8jZR2S-kJUT6GNjtXYU9hocNRiVUGZWvc/edit#gid=127492362)
+- Copy over any open test issues from previous release (ask previous CI Signal lead for the tracker sheet) and follow up on them with owners
+- Monitor [master-blocking](https://k8s-testgrid.appspot.com/sig-release-master-blocking) and [master-upgrade](https://k8s-testgrid.appspot.com/sig-release-master-upgrade) dashboards **twice a week** and ensure all failures and flakes are tracked via open issues
+  - Make sure all open issues have a kind/bug, priority/flake or priority/failing-test, priority/important-soon labels 
+  - Make sure the issue is assigned against the current milestone 1.x, using /milestone
+  - Assign the issue to appropriate SIG using /sig label
+  - CC the release manager and bug triage lead
+  - Add @kubernetes/sig-foo-test-failures to draw SIG-foo’s attention to the issue
+  - Post the test failure in SIG’s Slack channel to get help in routing the issue to the rightful owner(s)
```

Comment:
Also useful to note here that @mention of individuals associated with the issue and SIG leads tends to result in faster turn around.